app/views/admin/pages/_form.html.erb

```diff
@@ -3,15 +3,17 @@
 
   <div class="form-group">
     <%= form.label :page_type %>
-    <%= form.text_field :page_type, class: 'form-control' %>
-  </div>
-  <div class="form-group">
-    <%= form.label :published %>
-    <%= form.check_box :published, class: 'form-control' %>
+    <%= form.text_field :page_type, class: 'form-control', readonly: true %>
   </div>
   <div class="form-group">
     <%= form.label :content %>
     <%= form.text_area :content, class: 'form-control', 'data-provider': :summernote %>
   </div>
+  <div class="form-check">
+    <%= form.check_box :published, class: 'form-check-input' %>
+    <%= form.label 'Publish version to live site', class: 'form-check-label' %>
```

Comment:
@sameera207 can you put back the `:published` in the label so they match up? (Can still keep the custom label text). It matches the id and for attributes is all so you can click on the label